src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -861,15 +861,9 @@ Scene* svgSceneBuild(SvgLoaderData& loaderData, Box vBox, float w, float h, Aspe
         docNode->translate(-vBox.x, -vBox.y);
     }
 
-    auto viewBoxClip = Shape::gen();
-    viewBoxClip->appendRect(0, 0, w, h);
-
-    auto compositeLayer = Scene::gen();
-    compositeLayer->composite(std::move(viewBoxClip), CompositeMethod::ClipPath);
-    compositeLayer->push(std::move(docNode));
-
     auto root = Scene::gen();
-    root->push(std::move(compositeLayer));
+    root->push(std::move(docNode));
+    root->viewport(0, 0, static_cast<int32_t>(w), static_cast<int32_t>(h));
```

Comment:
Just curious...
Doesn't this affect the result of `bounds()`? I haven't tested it yet...
The result of `bounds()` after `CompositeMethod::ClipPath` and the result of `bounds()` after viewport are expected to be different. (Until at least one `update()` occurs...)

Reply:
@mgrudzinska  bounds() wouldn't occupy the compositors regions including clipper. Please double-check it.